### PR TITLE
[ticket/15548] Fix dead link in ACP_COOKIE_SETTINGS_EXPLAIN language entry

### PR DESCRIPTION
**Pull request by rxu**
Checklist:

- [x] Correct branch: master for new features; 3.2.x for fixes
- [x] Tests pass
- [x] Code follows coding guidelines: [master](https://area51.phpbb.com/docs/dev/master/development/coding_guidelines.html) and [3.2.x](https://area51.phpbb.com/docs/dev/3.2.x/development/coding_guidelines.html)
- [x] Commit follows commit message [format](https://area51.phpbb.com/docs/dev/3.2.x/development/git.html)

Tracker ticket (set the ticket ID to **your ticket ID**):

<a href="https://tracker.phpbb.com/browse/PHPBB3-15548">PHPBB3-15548</a>.

### CODE REVIEW

**Comment by DavidIQ** [comment]
This is an error in the site's configuration. The link is correct.

File: phpBB/language/en/acp/board.php [hunk 1]
```diff
@@ -348,7 +348,7 @@
 
 // Cookie Settings
 $lang = array_merge($lang, array(
-	'ACP_COOKIE_SETTINGS_EXPLAIN'		=> 'These details define the data used to send cookies to your users browsers. In most cases the default values for the cookie settings should be sufficient. If you do need to change any do so with care, incorrect settings can prevent users logging in. If you have problems with users staying logging in to your board, visit the <b><a href="https://www.phpbb.com/support/go/cookie-settings/">phpBB.com Knowledge Base - Fixing incorrect cookie settings</a></b>.',
+	'ACP_COOKIE_SETTINGS_EXPLAIN'		=> 'These details define the data used to send cookies to your users browsers. In most cases the default values for the cookie settings should be sufficient. If you do need to change any do so with care, incorrect settings can prevent users logging in. If you have problems with users staying logging in to your board, visit the <b><a href="https://www.phpbb.com/support/docs/en/3.1/kb/article/fixing-incorrect-cookie-settings/">phpBB.com Knowledge Base > Fixing incorrect cookie settings</a></b>.',
```
Comment (DavidIQ) [inline phpBB/language/en/acp/board.php]: The link should be: https://www.phpbb.com/support/go/cookie-settings (without the trailing slash)

Comment (rxu) [reply phpBB/language/en/acp/board.php]: Ah. Then let's just close the PR.

Comment (DavidIQ) [reply phpBB/language/en/acp/board.php]: Well I mean this pr should just remove that trailing slash I think.

Comment (rxu) [reply phpBB/language/en/acp/board.php]: lol. Got it.

**Comment by marc1706** [comment]
Merged as part of https://github.com/phpbb/phpbb/pull/5191

**Pull request closed without merging**